db/people.yml

```diff
@@ -636,6 +731,49 @@ cyshi:
       group: bg
   name: Chenyang Shi
   position: Scientist
+dpodgorski:
+  active: false
+  aka: []
+  avatar: https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y
+  bio: bio
+  education:
+    - begin_year: 0
```

Comment:
begin_year may not be required, in which case we should just delete teh whole line.  But if it is required, make it 2014 maybe